client/src/js/templates/dc/dc_autoproc.html

```diff
@@ -43,19 +43,19 @@
         <tbody>
             <tr>
                 <td>Start</td>
-                <td><%-BEAM.XBEAM%></td>
-                <td><%-BEAM.YBEAM%></td>
+                <td><%= BEAM.XBEAM ? BEAM.XBEAM : '- -' %></td>
```

Comment:
Make sure to use - and not =. - Makes sure the value is xss escaped

Reply:
FIxed